src/com/cdd/bao/template/Vocabulary.java

```diff
@@ -354,10 +354,8 @@ private void loadLabels(File baseDir, File[] extra, Set<String> exclude) throws
 		{
 			try 
 			{
-				if(f.getName().endsWith(".ttl")) //handles only .ttl files (Jason)
-					RDFDataMgr.read(model, f.getPath(), f.getName().endsWith(".ttl") ? Lang.TURTLE : Lang.RDFXML);
-				else if (f.getName().endsWith(".owl")) //handles .owl files (Jason)
-					RDFDataMgr.read(model, f.getPath());
```

Comment:
The change from a File to URL path is the only change that I made that didn't get merged.

Reply:
Check the master branch: I'm fairly sure the corrected version is in there:

			try 
			{
				URL fileURL = new File(f.getPath()).toURI().toURL(); // changing file to a URL for passing into Jena's RDF reader			
				RDFDataMgr.read(model, fileURL.getPath(), f.getName().endsWith(".ttl") ? Lang.TURTLE : Lang.RDFXML);
			}

Reply:
Yes, it's there on the remote. It's just my local master that didn't get the update yet... sorry for the confusion!